student.rb

```diff
@@ -1,4 +1,7 @@
+require_relative './high_five/'
 class Student
+  include HighFive
```

Comment:
There are things that are common among all three initial classes.  For Release 0 we could make a person class and pull up the common things.

```ruby
require_relative './high_five/'

class Person
  include HighFive

  attr_reader :age, :phase
  attr_accessor :name

  def initialize(options = {})
    @age = options.fetch(:age, 0)
    @name = options.fetch(:name, "")
  end
end
```